scripts/prepack.ts

```diff
@@ -34,13 +39,24 @@ export async function preparePackage({ sourcePackageDir }: { sourcePackageDir: s
 	// construct the final package.json
 	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	const newManifest = structuredClone({
-		// filter out comments
-		...Object.fromEntries(Object.entries(manifest).filter(([key]) => !key.startsWith('/*'))),
-		main: 'dist/index.js',
-		module: 'dist/index.mjs',
+		// filter out comments and `types`
+		...Object.fromEntries(Object.entries(manifest).filter(([key]) => !key.startsWith('/*') && key !== 'types')),
+		main: 'dist/cjs/index.js',
```

Comment:
```suggestion
		main: 'dist/cjs/index.cjs',
```

although, maybe we don't need the .cjs exetentsion at all if we have the .mjs extension for the esm files? We could just have the commonjs files as .js?

Reply:
Sure, but the explicit file extension is, well, explicit. There should be less confusion for builders, node, etc. with missing "type" values in `package.json` etc. that have been a part of the pain moving over to ESM bit by bit in various dependencies. If the file extension says `cjs` then there shouldn't be any magic or differing resolution required that may break. At least that was my logic here.